README.md

```diff
@@ -65,27 +64,13 @@ To watch and auto-compile it while working:
 $ stylus -w -o initial-setup.css initial-setup/main.styl
 ```
 
-## Screenshots
-
-![](https://raw.githubusercontent.com/auth0/wp-auth0/master/screenshot-1.png)
-
-![](https://raw.githubusercontent.com/auth0/wp-auth0/master/screenshot-2.png)
-
-![](https://raw.githubusercontent.com/auth0/wp-auth0/master/screenshot-3.png)
-
-![](https://raw.githubusercontent.com/auth0/wp-auth0/master/screenshot-4.png)
-
-![](https://raw.githubusercontent.com/auth0/wp-auth0/master/screenshot-5.png)
-
-![](https://raw.githubusercontent.com/auth0/wp-auth0/master/screenshot-6.png)
-
 ## Issue Reporting
 
-If you have found a bug or if you have a feature request, please report them at this repository issues section. Please do not report security vulnerabilities on the public GitHub issue tracker. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.
+If you find a bug or if you have a feature request, please report them in [Issues for this repo](https://github.com/auth0/wp-auth0/issues). Please do not report security vulnerabilities in a public GitHub issue tracker. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.
```

Comment:
I think it looks better this way:

...report them in the [issues tab](https://github.com/auth0/wp-auth0/issues)